src/HtmlProcessor/AbstractHtmlProcessor.php

```diff
@@ -320,10 +291,6 @@ private function normalizeDocumentType(string $html): string
      * Adds a Content-Type meta tag for the charset.
      *
      * This method also ensures that there is a HEAD element.
-     *
-     * @param string $html
-     *
-     * @return string the HTML with the meta tag added
```

Comment:
The main description says that the method will add the `meta` tag, so the description for the `@return` annotation is superfluous repetition.